utils/statistics-flow-reduce-operators.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+/**
+ * It's about reduction
+ *
+ * FROM:
+ *
+ * [date][serviceName].businessProcess[status] = num;
+ * [date][serviceName].certificate[name][status] = num;
+ * [date][serviceName].processingStep[stepName].pending.businessProcess = num;
+ * [date][serviceName].processingStep[stepName].pending.certificate[name] = num;
+ * [date][serviceName].processingStep[stepName].byProcessor[processorId][status].businessProcess =
+ * num;
+ * [date][serviceName].processingStep[stepName].byProcessor[processorId][status].certificate[name] =
+ * num;
+ *
+ * TO:
+ * [date][processorId] = {
+ *  processor: processorId,
+ *  date: "YYYY-MM-DD",
```

Comment:
This duplicates information which already thater and that makes result more heavy (we send it to client so better if response is light).

Additionally you have this information out of a box in a view, just use utils that make it friendly and readable:

```javascript
var toArray = require('es5-ext/object/to-array');
...
tbody(toArray(result, function (dateResult, date) {
  return toArray(dateResult, function (processorResult, processorId) {
    return tr(toArray(processorResult, function (data, key) {
      ...
    });
  });
}));
```